sycl/test-e2e/Graph/invalid_profiling.cpp

```diff
@@ -0,0 +1,160 @@
+// REQUIRES: level_zero || cuda, gpu
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out 2>&1
```

Comment:
Not sure if we need this IO redirection in the new tests
```suggestion
// RUN: %{run} %t.out
```